README.md

```diff
@@ -1,2 +1,228 @@
-# satellite6-upgrade
-Satellite 6 upgrade related tools
+# Satellite6-upgrade
+
+Satellite6 upgrade is a set of tools to be used for following reasons:
```

Comment:
`Satellite6 upgrade contains a set of tools that helps to perform various upgrade specific scenarios and moreover, it helps to validate data between pre and post upgrade to ensure data integrity post upgrade.` The highlights of Satellite6 upgrade are: